src/teama/ProductArrayList.java

```diff
@@ -15,83 +15,96 @@
  * @author Thomas
  *  */
 
-public class ProductArrayList implements Evaluation{
-	
-	//The actual reference that holds the data file and ArrayList of Products
-	
+public class ProductArrayList implements Evaluation {
+
+	// The actual reference that holds the data file and ArrayList of Products
+
 	private ArrayList<Product> productList;
 	private ProductDataFile pd;
 
-	//Constructor, ideally to put the data file into the ArrayList
-	
-	public ProductArrayList(){
-		
+	// Constructor, ideally to put the data file into the ArrayList
+
+	public ProductArrayList() {
+
 		pd = new ProductDataFile();
 		productList = pd.getPList();
-		
+
 	}
-	
-	//To check an item in the .txt file and see if it exists,
-	//if so return the item. If not, "No product has been found."
-	
+
+	// To check an item in the .txt file and see if it exists,
+	// if so return the item. If not, "No product has been found."
+
 	public Product lookUpProduct(String item) {
 		System.out.println("Looking up: " + item + " from" + productList.size() + " products");
-		for(Product p : productList) {
-		if(p.getName().equalsIgnoreCase(item))
-			return p;	
+		for (Product p : productList) {
+			if (p.getName().equalsIgnoreCase(item))
+				return p;
 		}
 
 		System.out.println("No product has been found.");
 		return null;
 
 	}
-	
-	//Look up an item in the .txt file and return the price of it
+
+	// Look up an item in the .txt file and return the price of it
 	public double lookUpItemPrice(String item) {
-	 	System.out.println("Looking up the price for: "  + item);
-		for(Product p : productList) {
-		if(p.getName().equalsIgnoreCase(item))
-			// 
-			return p.getPrice();			// TODO - fix the price both here and in ProductData class,
-										// may require modification in the Product class itself
+		System.out.println("Looking up the price for: " + item);
+		for (Product p : productList) {
+			if (p.getName().equalsIgnoreCase(item))
+				//
+				return p.getPrice(); // TODO - fix the price both here and in ProductData class,

```

Comment:
Here you can use method from above to lookUpProduct(), if found - return getPrice, not found -0.